src/scss/_variables.scss

```
@@ -0,0 +1,20 @@
+$blue:    #4072bd;
+$indigo:  #532896;
+$purple:  #705e93;
+$pink:    #852756;
+$red:     #6d1921;
+$orange:  #91582a;
+$yellow:  #c2a85a;
+$green:   #22db31;
+$teal:    #1d6b54;
+$cyan:    #51bdd3;
+
+$spacer: 1rem;
```

```suggestion
$spacer: 2rem;
```

Hey I thought this was meant to be 2, did you change your mind?

---

Nah it was always 1, you are wrong. 1 is best.